app/indexers/descriptive_metadata_indexer.rb

```diff
@@ -15,7 +19,7 @@ def to_solr
       'sw_language_ssim' => language,
       'mods_typeOfResource_ssim' => resource_type,
       'sw_format_ssim' => sw_format,
-      'sw_genre_ssim' => display_genre,
```

Comment:
Should the display_genre method be removed?

Reply:
yes thank you for the reminder!